src/main/java/org/commcare/core/graph/c3/AxisConfiguration.java

```diff
@@ -187,7 +191,7 @@ private void addTitle(JSONObject axis, String key, String position) throws JSONE
 
         // Show title regardless of whether or not it exists, to give all graphs consistent padding
         JSONObject label = new JSONObject();
-        label.put("text", title);
+        label.put("text", "title, rad");

```

Comment:
what does this do ?

Reply:
Ahh, I actually used it for testing, accidentally pushed it. Thanks for catching